package.json.sample

```diff
@@ -9,35 +9,35 @@
   },
   "homepage": "https://magento.com/",
   "devDependencies": {
-    "glob": "~7.1.1",
-    "grunt": "~1.0.1",
+    "glob": "~7.1.7",
+    "grunt": "~1.4.1",
     "grunt-autoprefixer": "~3.0.4",
     "grunt-banner": "~0.6.0",
     "grunt-continue": "~0.1.0",
-    "grunt-contrib-clean": "~1.1.0",
-    "grunt-contrib-connect": "~1.0.2",
-    "grunt-contrib-cssmin": "~2.2.1",
-    "grunt-contrib-imagemin": "~2.0.1",
-    "grunt-contrib-jasmine": "~1.2.0",
-    "grunt-contrib-less": "^2.1.0",
-    "grunt-contrib-watch": "~1.0.0",
-    "grunt-eslint": "~20.1.0",
+    "grunt-contrib-clean": "~2.0.0",
+    "grunt-contrib-connect": "~3.0.0",
+    "grunt-contrib-cssmin": "~4.0.0",
+    "grunt-contrib-imagemin": "~4.0.0",
+    "grunt-contrib-jasmine": "~3.0.0",
```

Comment:
This breaks unit and static tests, as it starts using chrome headless instead of phantom js. I tried to update it separately on the warden, and additional packages were required to be installed

Reply:
grunt-contrib-jasmine start from version 2 use headless chrome instead. Seem we need recheck old stuff if we want to upgrade